### DISCUSSION TITLE
donations/patreon/ko-fi ?

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#304] [General]
ik i've brought this up or maybe someone else did, but i was wondering if anyone but me would love to donate to this project ? i LOVEEE toonsquid and i feel donations would be helpful ? :D ofc if keiwan isn't okay w it then that would be okay :0 
![my-melody-my-melo](https://github.com/keiwando/toonsquid/assets/115050959/1da8883e-9621-4af3-8478-60a8c25ecd19)

dance dance

**Comment by DWrightAnimation** [comment]
I think donations would help speed up development of the app, but I also think people just paying for the app is helping the development as we speak. However, I think if more people worked on it, it'd make things easier on Keiwando.
